nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -133,7 +133,8 @@ def apply(
             for node in graph.get_all_nodes()
             if self._backend_entity.is_node_with_bias(node, graph)
         ]
-
+        if not node_and_bias_value:
+            return model
```

Comment:
It is not clear for me why this is required. Let's look:

- If `node_and_bias_value` is empty, the `for` statement is not executed.
- The `node_and_new_bias_value` is empty because the `for` statement was not executed.
- The `transformation_layout` is empty because the `node_and_new_bias_value` is empty
- The `model_transformer.transform(transformation_layout)` returns the same model because `transformation_layout` is empty

So, this special check is not required. Everything works as expected.

Reply:
This is because of the printing logs even in cases when  `node_and_bias_value` has no elements.

Reply:
I got it. But incorrect logging is not the problem of the FBC algorithm, is it? It should be fixed inside the `track` class.

https://github.com/openvinotoolkit/nncf/blob/50feea44ac6d705e7d0752e1f1b0f1f1bfb01c01/nncf/common/logging/track_progress.py#L50

Reply:
I plan to make changes to `track` class in the near future in order not to display progress bar for empty sequence.

Reply:
I will take the changes from this commit- https://github.com/nikita-savelyevv/nncf/commit/0485f6f1c3c089f5d031675222dcefbd70c2af9a and revert changes inside of FBC

Reply:
Done

Reply:
@kshpv @nikita-savelyevv Thanks!